montage.js

```diff
@@ -0,0 +1,64 @@
+const fs = require('fs');
+const gm = require('gm').subClass({imageMagick: true});
+const request = require('request');
+const rimraf = require('rimraf')
+
+const aws = require('./aws')
+
+const dir = './tmp';
```

Comment:
unlikely to be an issue, but if two users try making a montage at the same time there could be a collision and images would get overwritten. you could probably overwrite this by making the `dir = `./tmp/${id}` or something like that?